packages/squiggle-lang/src/rescript/FR/FR_Danger_Aggregate.res

```diff
@@ -0,0 +1,171 @@
+open FunctionRegistry_Core
+open FunctionRegistry_Helpers
+
+let nameSpace = "Danger_Aggregate"
+let requiresNamespace = true
+
+module AggregateFs = {
+  module Helpers = {
+    let checker = (fn, xs, minLength) => {
+      switch E.A.length(xs) < minLength {
+      | true =>
+        Error(
+          "Aggregation method does not make sense with fewer than" ++
+          Belt.Int.toString(minLength) ++ " elements",
+        )
+      | false => {
+          let checkedIndividualXs = E.A.fmap(x =>
+            switch (x, x > 1.0, x < 0.0) {
+            | (0.0, _, _) => Error("0 is not a probability")
+            | (1.0, _, _) => Error("1 is not a probability")
+            | (_, true, _) => Error("Probabilities can't be higher than 1")
+            | (_, _, true) => Error("Probabilities can't be lower than 0")
+            | (_, false, false) => Ok(x)
+            }
```

Comment:
I think at this point, I'd recommend trying to do this more neatly. This should probably be done with a few separate checks and cleanup, as opposed to a big switch statement, which is fairly hacky

Reply:
Tried to address in next version

Reply:
I suggest clicking "Resolve conversation" on items like this after they have been addressed (after the new code is submitted)